src/Framework/Doctrine/DoctrineSchemaSubscriber.php

```diff
@@ -11,8 +11,6 @@
 use Symfony\Component\DependencyInjection\Attribute\When;
 
 /**
- * @codeCoverageIgnore
```

Comment:
do not remove: https://docs.phpunit.de/en/11.4/annotations.html#codecoverageignore